web/src/components/menu.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react'
 
 import Navbar from 'react-bootstrap/Navbar'
+import Nav from 'react-bootstrap/Nav'
 import { Link } from 'react-router-dom'
 import PropTypes from 'prop-types'
 
 class TopMenu extends React.Component {
   render () {
+    const search = window.location.search
     return <Navbar bg="light" expand="lg">
-      <Navbar.Brand ><Link className="navbar-brand" to={`/r/${this.props.index}`}>Monocle</Link></Navbar.Brand>
+      <Navbar.Brand>
+        <Link className="navbar-brand" to={`/r/${this.props.index}`}>Monocle</Link>
+      </Navbar.Brand>
+      <Nav className="mr-auto">
+        <Link className="nav-link" to={`/${this.props.index}${search}`}>Top</Link>
+        <Link className="nav-link" to={`/${this.props.index}/merged-changes${search}`}>Merged</Link>
```

Comment:
What about a "dropdown menu" for "Changes" that contains "Merged Changes" and "Open Changes" (https://getbootstrap.com/docs/4.0/components/navbar/) ?

Reply:
I think we'll switch to dropdown when we'll have more entries.

Reply:
Ok so I merge :) !